python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -164,20 +164,21 @@ def _set_ecs_tags(self, run_id, task_arn):
         except ClientError:
             pass
 
-    def _set_run_tags(self, run_id, task_arn):
+    def set_run_tags(self, run_id, task_arn):
+        # Extracted method to allow for subclasses to customize the ECS tag storage implementation
         cluster = self._task_metadata().cluster
         tags = {"ecs/task_arn": task_arn, "ecs/cluster": cluster}
         self._instance.add_run_tags(run_id, tags)
 
-    def _get_run_tags(self, run_id):
+    def get_ecs_info(self, run_id) -> EcsInfo:
```

Comment:
bit confused why we took out the underscore - them being overridden by a subclass doesn't necc make them public I think?

Reply:
def _get_ecs_info_from_run_tags?